docs/deprecations.md

```diff
@@ -17,13 +17,14 @@ being deprecated.
+| [The `PipelineRun.Spec.ServiceAccountNames` field is deprecated and will be removed.](https://github.com/tektoncd/pipeline/issues/2614)                             | [v0.15.0](https://github.com/tektoncd/pipeline/releases/tag/v0.15.0) | Beta                                                                                                   | May 15 2021                         |
+| [`Conditions` CRD is deprecated and will be removed. Use `when` expressions instead.](https://github.com/tektoncd/community/blob/main/teps/0007-conditions-beta.md) | [v0.16.0](https://github.com/tektoncd/pipeline/releases/tag/v0.16.0) | Alpha                                                                                                  | Nov 02 2020                         |
+| [The `disable-home-env-overwrite` flag will be removed](https://github.com/tektoncd/pipeline/issues/2013)                                                           | [v0.24.0](https://github.com/tektoncd/pipeline/releases/tag/v0.24.0) | Beta                                                                                                   | February 10 2022                    |
+| [The `disable-working-dir-overwrite` flag will be removed](https://github.com/tektoncd/pipeline/issues/1836)                                                        | [v0.24.0](https://github.com/tektoncd/pipeline/releases/tag/v0.24.0) | Beta                                                                                                   | February 10 2022                    |
+| [The `scope-when-expressions-to-task` flag will be flipped from "false" to "true"](https://github.com/tektoncd/pipeline/issues/1836)                                | [v0.27.0](https://github.com/tektoncd/pipeline/releases/tag/v0.27.0) | Beta                                                                                                   | February 10 2022                    |
+| [The `scope-when-expressions-to-task` flag will be removed](https://github.com/tektoncd/pipeline/issues/1836)                                                       | [v0.27.0](https://github.com/tektoncd/pipeline/releases/tag/v0.27.0) | Beta                                                                                                   | March 10 2022                       |
+| [`PipelineResources` are deprecated.](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)                                     | [v0.30.0](https://github.com/tektoncd/pipeline/releases/tag/v0.30.0) | Alpha                                                                                                  | Dec 20 2022                         |
```

Comment:
since these are alpha i think technically we could remove them 1 month after we announce - so Dec 2021 or Jan 2022

(we definitely won't however)

Reply:
good catch! meant to say 2021 (already in 2022 with all the planning we've been doing for next year 😃)

Reply:
THATS BEEN HAPPENING TO ME TOO!!!